utilities/functions.py

```diff
@@ -53,3 +86,69 @@ def get_corners_from_contour(contour: np.ndarray, corner_number=4) -> None:
             upper = current
         i += 1
     return hull
+
+
+# The next two functions' `x` and `y` arguments are in camera coordinates.
+# The origin is the top left corner of the screen, with left and down being positive.
+
+# Both have an `invert` argument which, if False, leaves the return angle with negative
```

Comment:
"negative at the origin" and "positive at the origin" are oxymorons. They are 0 at the origin, by definition. This comment does not explain what you mean.

Reply:
I have clarified this.